build/stf-collect-logs/tasks/main.yml

```diff
@@ -1,35 +1,23 @@
 ---
-- name: "Get builds"
-  ansible.builtin.shell:
-    cmd: |
-      echo "*** [INFO] Showing oc get builds" > {{ logfile_dir }}/post_oc_get_builds.log 2>&1
-      oc -n {{ namespace }} get builds -oyaml >> {{ logfile_dir }}/post_oc_get_builds.log 2>&1
-      echo "*** [INFO] Showing oc get builds -oyaml" >> {{ logfile_dir }}/post_oc_get_builds.log 2>&1
-      oc -n {{ namespace }} get builds -oyaml >> {{ logfile_dir }}/post_oc_get_builds.log 2>&1
-      cat {{ logfile_dir }}/post_oc_get_builds.log
+- name: "Get resources logs [ Builds, Subscriptions, Image, Imagestreams, Pods ]"
+  ansible.builtin.shell: |
+    for resource in {{ resource_types|join(' ') }}; do
+      log_file="{{ logfile_dir }}/post_oc_get_$resource.log"
+      echo "*** [INFO] Showing oc get '$resource'" > "$log_file" 2>&1
+      oc -n {{ namespace }} get "$resource" >> "$log_file" 2>&1
+      echo "[INFO] oc get '$resource' -oyaml" >> "$log_file" 2>&1
+      oc -n {{ namespace }} get "$resource" -oyaml >> "$log_file" 2>&1
+    done
+  delay: 10
```

Comment:
This is not required unless ``retries`` is also set.

https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_loops.html#retrying-a-task-until-a-condition-is-met